CHANGELOG.md

```diff
@@ -1,5 +1,54 @@
 # CHANGELOG
 
+## v1.10.2
+
+*August 14th, 2024*
+
+This release brings significant performance improvements and introduces a new configuration options for better control over packet clearing. Enhancements include faster startup times through optimized queries and the introduction of a `clear_limit` setting for packet clearing. Additionally, bug fixes and updates to the integration test framework ensure greater stability and compatibility across various environments.
+
+### BUG FIXES
+
+- [Relayer Library](relayer)
+  - Fix the `memo_overwrite` configuration to correctly apply the
+    overwrite if it is configured.
+    ([\#4104](https://github.com/informalsystems/hermes/issues/4104))
+- [Telemetry & Metrics](telemetry)
+  - Fix the `dynamic_gas_queried_success_fees` Prometheus metric name.
+    ([\#4104](https://github.com/informalsystems/hermes/issues/4104))
+    
+
+### IMPROVEMENTS
+
+- [Integration Test Framework](tools/test-framework)
```

Comment:
Let's put this at the bottom, since users are most interested in improvements to the relayer itself.

Reply:
Done in 53b7fdf6af227c26b3c38ca8c939ef3e35d093ee